coalib/bearlib/languages/definitions/XML.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class XML:
+    extensions = '.xml', '.xsl', '.rss', '.xslt', '.xsd', '.wsdl', '.wsf'
```

Comment:
see http://api.coala.io/en/latest/coalib.bearlib.languages.html#module-coalib.bearlib.languages.Language we can derive XSL etc. from XML